websites_schema.yml

```diff
@@ -12,21 +12,22 @@ mapping:
             unique: yes
           "url":
             type: str
-            pattern: /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/
+            pattern: /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,8}\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)/
```

Comment:
For future reference, Dan's Tools Regexpal is great for checking regex correctness: https://www.regexpal.com/

May be a good idea to run some strings through this when modifying/updating patterns.

Reply:
thanks for that. we had a .software url that was failing